### ISSUE TITLE
Arbitrary task properties

### ISSUE THREAD

**Issue by justusthane** [tgrosinger/tq-obsidian#18]
**Is your feature request related to a problem? Please describe.**
Not really a problem, but I like to give some tasks a "start on" date in addition to a due date, when tasks aren't relevant until after a certain date.

To allow for this, as well as other specific needs, it might be nice to allow for arbitrary custom properties.

**Describe the solution you'd like**
It would be great to be able to enter arbitrary custom properties in the task note frontmatter, and then query those properties from a tq code block.

**Comment by tgrosinger** [comment]
I agree. I already have a TODO for this in the code, but haven't gotten to it because I haven't encountered the need yet.

When you imagine this feature, do you think this would be something where you configure the extra fields in the settings and it applies every time you create a task? Or would you want to create the extra fields each time? I would lean towards the first, otherwise typos seem likely.

**Comment by justusthane** [comment]
Yes, probably the former.
